src/handlers/token.ts

```diff
@@ -67,6 +67,7 @@ export const generateToken = (tokenUser: TokenUser | TokenUserL2) =>
             config.JWT_TOKEN_PRIVATE_KEY,
             tokenUser,
             tokenExpiration,
+            config.JWT_TOKEN_KID,
             config.JWT_TOKEN_ISSUER
```

Comment:
the order of this prop should be inverted.

Reply:
Fixed by https://github.com/pagopa/hub-spid-login-ms/pull/37/commits/64ecccb1847ef33f6e8c4f8e703259801c0c6755

Reply:
We should investigate why the unit test complete successfully instead of returning a failure

Reply:
because token.ts didn't have unit test